docs/receipts.md

```diff
@@ -0,0 +1,42 @@
+# Document Generation in Indico
+
+Indico includes a module for generating documents, such as receipts and certificates, from templates that can be defined in the event or category. This feature is especially useful for conference organizers and administrators who need to provide attendees with customized documents quickly and efficiently.
+
+![](assets/document_templates/receipt-example.png)
+
+### Configuration
+
+Creating document templates requires knowledge of HTML and CSS and for security reasons is restricted to Indico admins. However, additional trusted users may be granted rights to create and edit templates
+In the _Document Templates_ section of the Indico administration page:
+
+![](assets/document_templates/configuring.png)
+
+### Creating a template
+
+Indico lets you define templates for various types of documents, including receipts, certificates, and more. These templates allow you to specify the layout, content, and placeholders for dynamic data using HTML with Jinja2 templates.
+
+Templates can be either created under an event, or a category, making them available for use in all of its events. These can be viewed, added, cloned and deleted from the management area under the _Customisation_ tab on events, or on its own _Document Templates_ tab on categories.
+
+![](assets/document_templates/edit.png)
+
+When creating a new template you must specify its name, and optionally define a default filename. Then you can design your template using the code editor. You can write HTML with Jinja2 templates, CSS and specify custom variables in YAML.
+
+![](assets/document_templates/create.png)
+
+### Generating Documents
+
+Once there are templates defined in your event, Indico can automatically populate them with dynamic data, such as attendee names, event details, and registration information. All of this can be done in bulk for a large number of registrations.
+
+To do so, open the registrations list for your event, select the registrations for which you want to generate. Then, open the document generation dialog by clicking on _Generate Documents_ under _Actions_.
+
+![](assets/document_templates/generate.png)
+
+On this dialog you can select the template you want to use and set the relevant template parameters:
+
+![](assets/document_templates/generate2.png)
+
+Additionally, you have the option to _Publish document_. If selected, the document will be visible for the participant on their registration details page. Otherwise, it will only be visible for event managers, although it can be published at any moment.
```

Comment:
Is there actually a way to bulk publish these documents or would I have to regenerate them with the publish option checked?

Reply:
No, only bulk download. Not sure if bulk publishing makes much sense tbh - if you don't publish them it's probably because you want to print them yourself...

UI for it would also be tricky since you can have any amount of documents for a registrant/template...

Reply:
Yep agreed that it's probably not useful, this was more about the wording in the docs which to me suggests that I can publish the documents after they've been generated which is not actually the case (unless I do it one by one)